### DISCUSSION TITLE
Move db files to other directory

### DISCUSSION THREAD

**Discussion by wizwin** [Chia-Network/chia-blockchain#6199] [Ideas]
.chia folder contains the db files which are growing in size.

blockchain_v1_mainnet.sqlite - 3.8GB
blockchain_wallet_v1_mainnet.sqlite - 1.98GB
Total as of today - 5.78GB

Since chia does not allow selection of installation directory or where the DB has to reside, it is occupying lots of space on my boot SSD (which is small) - Windows (C:\) partition

Need a way to move it to a custom location (Not sure if I can symlink them)

**Comment by tiny0little** [comment]
I have not tried what you are about to do, but it seems that paths to node and wallet DBs are in the config file
```
$ cat ~/.chia/mainnet/config/config.yaml | egrep "db/"
  database_path: db/blockchain_v1_CHALLENGE.sqlite
  peer_db_path: db/peer_table_node.sqlite
  simulator_database_path: sim_db/simulator_blockchain_v1_CHALLENGE.sqlite
  simulator_peer_db_path: sim_db/peer_table_node.sqlite
  database_path: wallet/db/blockchain_wallet_v1_CHALLENGE_KEY.sqlite
  wallet_peers_path: wallet/db/wallet_peers.sqlite
```

similar questions
- https://github.com/Chia-Network/chia-blockchain/discussions/5356
- https://github.com/Chia-Network/chia-blockchain/discussions/2671

**Reply by wizwin** [reply]
@tiny0little Thank you. It would be nice to have the installer ask for the path.

**Comment by umitki** [comment]
> .chia folder contains the db files which are growing in size.
> 
> blockchain_v1_mainnet.sqlite - 3.8GB blockchain_wallet_v1_mainnet.sqlite - 1.98GB Total as of today - 5.78GB
> 
> Since chia does not allow selection of installation directory or where the DB has to reside, it is occupying lots of space on my boot SSD (which is small) - Windows (C:) partition
> 
> Need a way to move it to a custom location (Not sure if I can symlink them)


That was a real problem for me too, I made this steps and it worked! You can do this steps at the below link.

https://spacefarmers.io/wiki/guides/farming/movedb

**Reply by sntpham** [reply]
I have tried this on RasPi4 by changing the path to /mnt/XYZ/mainnet/db, but it did not work.
